data/sensors_recent.php

```diff
@@ -371,21 +348,20 @@ function output_cell($rowspan, $data) {
 
 				if ($row['firmware_version'] === null)
 					output_cell($rowspan, '< v1');
-				else if ($row['firmware_version'] !== 255)
+				else if ($row['firmware_version'] !== 255 && $row['firmware_version'] !== '255')
 					output_cell($rowspan, '<a href="https://github.com/meetjestad/mjs_firmware/tree/v' . $row['firmware_version'] . '">v' . $row['firmware_version'] . '</a>');
 				else
 					output_cell($rowspan, 'v' . $row['firmware_version']);
+
 				if ($row['latitude'] == '0.0' && $row['longitude'] == '0.0') {
 					output_cell($rowspan, 'No position');
 				} else {
 					$url = "http://www.openstreetmap.org/?mlat=" . $row['latitude'] . "&amp;mlon=" . $row['longitude'];
 					output_cell($rowspan, "<a href=\"" . $url . "\">" . $row["latitude"] . " / " . $row["longitude"] . "</a>");
 				}
 
-				if (array_key_exists('counter', $message)) {
-					output_cell($rowspan, $message['counter']);
-				} else if (array_key_exists('f_cnt', $message['uplink_message'])) {
-					output_cell($rowspan, $message['uplink_message']['f_cnt']);
+				if (array_key_exists('counter', $metadata)) {
```

Comment:
Volgens mij kan deze if weg, aangezien `$metadata['counter']` bovenaan nu altijd een waarde krijgt. Ik heb ook nog even in de database gechecked, maar zo te zien hebben *alle* ttn.v1 en ttn.v2 berichten ook een counter, dus ik weet niet zo goed waarom deze if er in de eerste plaats was. Dus kan nu zeker weg (en de `-` in de else dus ook).

Reply:
Aangepast.